test/spec/vis/layers-factory.spec.js

```diff
@@ -145,6 +150,50 @@ describe('vis/layers-factory', function () {
         });
       });
     });
+
+    describe('Is high resolution screen', function () {
+      beforeEach(function () {
+        spyOn(LayersFactory, 'isRetina').and.returnValue(true);
+      });
+
+      describe('urlTemplateRetina is not defined', function () {
+        it("should not convert to '" + urlTemplateRetina + "'", function () {
```

Comment:
IMO there is no need for the describe, I would use a simple `it('should use the default url template when there is not urlTemplateRetina defined');`